climada/hazard/centroids/centr.py

```diff
@@ -110,7 +110,7 @@ class Centroids():
     def __init__(self):
         """Initialize to None raster and vector"""
         self.meta = dict()
-        self.geometry = gpd.GeoSeries()
+        self.geometry = gpd.GeoSeries(crs=DEF_CRS)
```

Comment:
I'm having a quick look to see if this might have unexpected consequences anywhere. Could you check out `Centroids.set_vector_file`? I think a default CRS means you now can't read in vector files without changing the empty `geometry`'s CRS to match the file, which is very unintuitive.

Reply:
Good question. But to: "I think a default CRS means you now can't read in vector files without changing the empty geometry's CRS to match the file, which is very unintuitive." Then we should simply change the `read_vector_files` method right?

Reply:
But I can remove it again.

Reply:
@emanuel-schmid : do you think it is a good idea to a default CRS when initiating a `Centroid`?

Reply:
The main reason why I added a default CRS is for the method `Centroids.equal` which looks for equal CRS. If `Centroids.crs` is None, then `Centroids.equal(Centroids2)` will give an error 

```
  File "/climada_python/climada/hazard/centroids/centr.py", line 562, in union
    if not u_coord.equal_crs(cent.crs, self.crs):
  File "/climada_python/climada/util/coordinates.py", line 1324, in equal_crs
    return rasterio.crs.CRS.from_user_input(crs_one) == rasterio.crs.CRS.from_user_input(crs_two)
  File "anaconda3/envs/climada_210/lib/python3.8/site-packages/rasterio/crs.py", line 498, in from_user_input
    raise CRSError("CRS is invalid: {!r}".format(value))
rasterio.errors.CRSError: CRS is invalid: None

Reply:
So, just to make it simple, I removed the default CRS and added it if needed in the Union part. 
a8be18d

Reply:
> 
> 
> @emanuel-schmid : do you think it is a good idea to a default CRS when initiating a `Centroid`?

Actually I do. While we don't have to insist that users define a CRS, it makes things easier when a geometry object always has a well defined CRS nevertheless. 
Apart from that, Exposures do have a default CRS as well, so it supports consistency within CLIMADA.

Reply:
I think a default CRS is a good idea too, and I liked the implementation. My comment here was to point out that this method probably needs updating so that it can still read a file with a non-default CRS onto an empty Centroids object.

Reply:
I re-added the default crs here:
9c646d4